src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,48 +1,81 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import { Loader } from '../Loader';
+import { Todo } from '../../types/Todo';
+import { getUser } from '../../api';
+import { User } from '../../types/User';
+
+type Props = {
+  selectedTodo: Todo | null,
+  closeModal: () => void,
+};
+
+export const TodoModal: React.FC<Props> = ({
+  selectedTodo,
+  closeModal,
+}) => {
+  const [currentUser, setCurrentUser] = useState<User | null>(null);
+
+  const getUsersList = async () => {
```

Comment:
Hm, are you sure that it's user list? Because you use `getUser` here (not `getUsers`)